codes/swift/Package.swift

```diff
@@ -0,0 +1,21 @@
+// swift-tools-version: 5.7
+
+import PackageDescription
+
+let package = Package(
+    name: "HelloAlgo",
+    platforms: [
+        .macOS(.v12),
```

Comment:
Is this config only available on macOS? If so, could we make it work on Win and Linux also?